python/cxx/eclipse_io.cpp

```diff
@@ -463,13 +466,15 @@ void python::common::export_IO(py::module& m) {
         .def("units", &ESmryBind::units);
 
    py::class_<Opm::EclIO::EGrid>(m, "EGrid")
-        .def(py::init<const std::string &>())
+        .def(py::init<const std::string &, const std::string &, const bool>(), py::arg("filename"),
+             py::arg("grid_name") = "global", py::arg("apply_mapaxes") = true)
```

Comment:
I set this value to true following how this is done in `resdata`:

https://github.com/equinor/resdata/blob/main/python/resdata/grid/rd_grid.py#L312

Reply:
>> The default value for apply_mapaxes should still be false here.
>
> I set this value to true following how this is done in resdata:

Resdata's default settings should not be guiding us here.  In OPM, if someone wants MAPAXES they should have to be **very** explicit about it since MAPAXES is usually entirely optional and never impacts anything.